packages/cli/src/networks/index.ts

```diff
@@ -102,13 +101,20 @@ export function getGenesisFileUrl(network: NetworkName): string | null {
  * Fetches the latest list of bootnodes for a network
  * Bootnodes file is expected to contain bootnode ENR's concatenated by newlines
  */
-export async function fetchBootnodes(network: NetworkName): Promise<string[]> {
+
+class FetchBootFileError extends Error {}
```

Comment:
Should I move to ? https://github.com/ChainSafe/lodestar/blob/b5712a60637f6b13b7f60c6dfcadc214a8c71bf7/packages/utils/src/errors.ts#L13